marklogic-data-hub-central/ui/src/components/query-export/query-export-modal/query-export-modal.test.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { fireEvent, render } from "@testing-library/react";
+import QueryExportModal from './query-export-modal';
+
+
+describe('Query Export Component', () => {
+
+  let columns = ['id', 'firstName', 'lastName', 'age']
+
+  test('Verify Query Export Modal Dialog renders', () => {
```

Comment:
Some of the assertions in this test is being tested again in test 4 - line 29. I will again getting rid of this test and asserting for elements within functional tests which really test something.

This test is only asserting checkboxes are present in the dom. Line 14 from this test is repeated in line 37 (test 4). So if "All" is missing in the dom both the test will fail with same reason and it adds to the maintenance of fixing same failures in multiple places. If there was a different action being performed to assert this, then it would make sense. Otherwise this is just duplicative.

Reply:
Thanks Srikanth, I added fix in the pr 3937.